### PR TITLE
Arun's Week 7 Drop Down Changes and Svg Additions

### PR DESCRIPTION
**Pull request by abhattasali**
Modified Nav.js to Add Drop Down for settings and help pages; imported svg icons Help and Settings (modified); Settings.js development

### CODE REVIEW

File: src/components/containers/Nav.js [hunk 1]
```diff
@@ -19,7 +26,45 @@ const Header = styled.h1`
...
+    border: 2px solid #ddd;
+    border-radius: 4px;
+    color: #000;
+    transition: opacity 1s ease;
+  }
+
+  // &:hover .content {
+  //   visibility: visible;
+  //   height: auto;
+  // }
```
Comment (alexlongerbeam) [inline src/components/containers/Nav.js]: Remove these before merging

File: src/components/containers/Nav.js [hunk 2]
```diff
@@ -28,9 +73,32 @@ const Navbar = ({ onAdd }) => {
...
-      <Link to="/login">
-        <Icon src="menu.svg" alt="Menu" />
-      </Link>
+      
+      
+      
+      <Dropdown closed={closed} onClick={()=>setClosed(!closed)}>
+        <Icon src="settings.svg" alt="Menu" />
+     
+
```
Comment (alexlongerbeam) [inline src/components/containers/Nav.js]: Clean up all the extra lines here

**Review by alexlongerbeam — CHANGES_REQUESTED** [review]
Overall the code looks good! Can you address my comments and also add some screenshots/screen recordings to the Pull Request to basically show proof that things look as expected?

**Comment by abhattasali** [comment]
Made edits to original PR.

**Review by alexlongerbeam — APPROVED** [review]
Looks awesome!

**Pull request merged**